tests/src/common/Wsk.scala

```diff
@@ -354,7 +354,7 @@ class WskRule(override val usePythonCLI: Boolean = false)
             { if (enable) Seq("--enable") else Seq() } ++
             { shared map { s => Seq("--shared", if (s) "yes" else "no") } getOrElse Seq() }
         val result = cli(wp.overrides ++ params, expectedExitCode)
-        assert(result.stdout.contains("ok:"), result)
+        if (expectedExitCode == SUCCESS_EXIT) assert(result.stdout.contains("ok:"), result)
         if (enable) {
             val b = waitfor(() => checkRuleState(name, active = true), totalWait = 30 seconds)
```

Comment:
> i had to add that if guard to make the tests pass. are you saying that upstream/master no longer requires this?

Then we have a problem. The predicate and poll loop is waiting for rule to be enabled but create has finished implying the poll is not necessary. Or did I miss something?